tests/image_test.py

```diff
@@ -1225,5 +1226,20 @@ def test_load_img(self, tmpdir):
                                        interpolation="unsupported")
 
 
+    def test_list_pictures(self, tmpdir):
```

Comment:
This test is not idempotent. Is very unlikely but it could be that the test passes. Is better to define some cases that we need to check and use those ones instead of a random generation of filenames.

If this is a test for `list_pictures` is better to not use other functions in the test like `array_to_img` and `save`.

Reply:
The ```save``` is from ```Pillow``` not ```utils```.

What where you thinking of and how many?
```test.png```, ```test0.png```, ```test-1.png```, ```1test.jpg```, ```1-test.jpg``` enough?

Reply:
`-` at the end? I think does look good.